sqlmesh/core/context.py

```diff
@@ -1172,6 +1175,7 @@ def plan_builder(
         categorizer_config: t.Optional[CategorizerConfig] = None,
         enable_preview: t.Optional[bool] = None,
         run: bool = False,
+        rendered_model_diff: t.Optional[bool] = None,
```

Comment:
Since we currently don't expose this in config having this to be just bool with default `False` is fine.